packages/choose.rb

```diff
@@ -2,30 +2,23 @@
 
 class Choose < Package
   description 'make choices on the command line'
-  homepage 'https://github.com/geier/choose'
-  version '0.1.0'
+  homepage 'https://github.com/geier/choose/'
+  @_ver = '0.1.0'
+  version @_ver + '-1'
   license 'MIT'
   compatibility 'all'
-  source_url 'https://github.com/geier/choose/archive/v0.1.0.tar.gz'
-  source_sha256 'd09a679920480e66bff36c76dd4d33e8ad739a53eace505d01051c114a829633'
+  source_url 'https://github.com/geier/choose.git'
+  git_hashtag 'v' + @_ver
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/choose/0.1.0_armv7l/choose-0.1.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/choose/0.1.0_armv7l/choose-0.1.0-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/choose/0.1.0_i686/choose-0.1.0-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/choose/0.1.0_x86_64/choose-0.1.0-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '26cdefb57b3e12c1a73626879f0327aecc474289be086034c3f930d4cc51528b',
-     armv7l: '26cdefb57b3e12c1a73626879f0327aecc474289be086034c3f930d4cc51528b',
-       i686: 'e3db27cee1199945140e4524b9b2b4120d169190e37545015aed2cdd125def95',
-     x86_64: '138d368f47682787e47e01a59090014d993e8f63ba6678a9b315640a5075cd88',
-  })
+  depends_on 'py3_urwid'
+  depends_on 'py3_setuptools' => :build
 
-  depends_on 'setuptools'
+  def self.build
+    system "sed -i 's:#!/usr/bin/env python:#!#{CREW_PREFIX}/bin/python3:' choose"
+  end
 
   def self.install
-    system "pip install urwid --prefix #{CREW_PREFIX} --root #{CREW_DEST_DIR}"
-    system "install -Dm755 choose #{CREW_DEST_PREFIX}/bin/choose"
+    FileUtils.mkdir "#{CREW_DEST_PREIFX}/bin/"
```

Comment:
I can see you haven't tested. :) You will need the `_p` added to recursively create all directories.   Also `CREW_DEST_PREIFX` is misspelled.  The correct command should be:
`FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"`

Reply:
Yes, I haven't tested any of this. Naturally I will test everything before I open this up for review.